File: Data Structures & Algorithms/two-integer-sum/submission-0.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int i1 , j1;
        for(int i=0 ; i < nums.size() ; i++){
            for(int j = i+1 ; j< nums.size() ; j++){
                if(nums[i]+nums[j] == target){
                   i1 =i;
                   j1 =j;
                   return {i1, j1};
                }
            }
        }
    }
};

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.py
class Solution:
    def productExceptSelf(self, nums: List[int]) -> List[int]:
        pr = 1
        prz=1
        cz = 0
        for i in nums:
            pr*=i
            if i == 0:
                cz+=1
        if cz == 1:
            for i in nums:
                if i!=0:
                    prz*=i   
        if cz == 0:
            for i in range(len(nums)):
                nums[i]=int(pr/nums[i])
        elif cz == 1:
            for i in range(len(nums)):
                if nums[i]==0:
                    nums[i]=prz
                else:
                    nums[i]=0
        else:
            for i in range(len(nums)):
                nums[i]=0

        return nums